{{ cookiecutter.package_name }}/.travis.yml

```diff
@@ -63,22 +63,17 @@ env:
         # - FLAKE8_OPT='--ignore=E501'
         - FLAKE8_OPT=''
 
+    matrix:
```

Comment:
Why move this up? Just curious.

Reply:
some leftover from the rebase. I agree that it shouldn't be up here.